src/connection/connection_v1.ts

```diff
@@ -1,10 +1,17 @@
 import { ACCOUNT, ACCOUNT_BY_NAME } from "../common/api";
-import { AccountInfo, Connection as BaseConnection } from "./base";
+import { Connection as BaseConnection } from "./base";
 import { ResourceManager } from "../service";
 
 const INFRA_VERSION = 1;
 
+export interface AccountInfo {
+  id: string;
+  infraVersion: number;
```

Comment:
Can we remove the infraVersion from here? I believe we only have account_id in fb1.0

Reply:
Fair point, removed.